### DISCUSSION TITLE
Integration for Shapefile and Geodatabase files

### DISCUSSION THREAD

**Discussion by tjarross** [organicmaps/organicmaps#7045] [Ideas]
Hello,

I'm new in Organic Maps, and I would like to know if it is a good idea to implement imports for both shapefile and geodatabase files ?

It could be great for hikers to have more data for their trips.

I'm particularly thinking of using these maps for my personal projets: https://www.hydrosheds.org/hydroatlas

**Comment by sierracircle** [comment]
I second this idea.  In the past I have managed to find shapefiles for public-lands in the U.S (BLM, National Parks, etc)  and import them into OSM+, but it was a lot of effort on my part.
It would be great to have a simple way to import these type of files and share them.
Having BLM data on a map when hiking is a very nice feature and keeps me from stumbling onto private land when I am out in the wild.

**Comment by biodranik** [comment]
Is it about drawing some polygons on top of the map? #1491

**Reply by sierracircle** [reply]
For example:  Michigan State  Non-motorized trails  are offered in several formats:
https://gis-midnr.opendata.arcgis.com/datasets/5373c26cc353475fa64bd8094af763c7_20/explore?location=42.494052%2C-83.528955%2C9.39

Shapefile, KML and GeoJSON

Can any of those easily be imported into Organic Maps?  It would provide a trail-overlay and be very useful for back-country hikes.

**Comment by biodranik** [comment]
KML, KMZ and GPX import already works. Importing tracks and points already works. Only polygons/areas are not supported yet.
